modules/gitops-creating-user-defined-cluster-roles-and-configuring-user-defined-permissions-for-application-controller.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-creating-user-defined-cluster-roles-and-configuring-user-defined-permissions-for-application-controller_{context}"]
+= Creating user-defined cluster roles and configuring user-defined permissions for Application Controller
+
+As a cluster administrator, to add user-defined permissions to your aggregated cluster role, you must create one or more user-defined cluster roles and configure user-defined permissions for Argo CD Application Controller of a cluster-scoped Argo CD instance.
```

Comment:
```suggestion
Add user-defined permissions to your aggregated cluster role by creating one or more user-defined cluster roles then configure the user-defined permissions for the Argo CD Application Controller component of a cluster-scoped Argo CD instance.
```
I see "as a cluster administrator" throughout this PR where it isn't needed; this should be simply listed as is a prereq.

Reply:
I see no harm in stating the audience/persona of the content clearly at the start of the document.
To keep it consistent with what we already have in the GitOps docs, I prefer to retain this info as is.